tests/cxx/show_comments-d1.h

```diff
@@ -0,0 +1,23 @@
+//===--- show_comments-d1.h - test input file for iwyu --------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// this file exists to verify show_comments works
+#ifndef SHOW_COMMENTS_D1_H
+#define SHOW_COMMENTS_D1_H
+
+#include "tests/cxx/show_comments-d2.h"  // Transitive include
```

Comment:
Ah, if you want to do transitive includes, that header should be called `-i1.h`, `i` for "indirect"